sw/airborne/modules/computer_vision/viewvideo.c

```diff
@@ -197,7 +205,7 @@ void viewvideo_init(void)
 
 #if VIEWVIDEO_USE_NETCAT
   // Create an Netcat receiver file for the streaming
-  sprintf(save_name, "%s/netcat-recv.sh", STRINGIFY(VIEWVIDEO_SHOT_PATH));
```

Comment:
But only because it is already a string, i.e. the default?
What about if you add `<define name="VIEWVIDEO_SHOT_PATH" value="/foo/bar"/>` to your airframe file to set a custom location?